### PR TITLE
update role yaml configmap group to empty

### PR DESCRIPTION
**Pull request by wackxu**
We should follow the principle of least privilege, so update all group to k8s core group

### CODE REVIEW

File: install/kubernetes/helm/istio-remote/charts/sidecarInjectorWebhook/templates/clusterrole.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ metadata:
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
 rules:
-- apiGroups: ["*"]
```
Comment (gyliu513) [inline install/kubernetes/helm/istio-remote/charts/sidecarInjectorWebhook/templates/clusterrole.yaml]: Please update the same file in `istio` as well.

**Comment by wackxu** [comment]
@gyliu513 Update, PTAL

**Pull request merged**